_data/projects/log-agent.yml

```
@@ -0,0 +1,8 @@
+name: LogAgent
+desc: LogAgent is java deamon process that periodically sends running log files to end point storage such as AWS S3, GCP GCS etc.
+site: https://github.com/sagar-arora/LogAgent
+tags:
+  -good first issue
```

I think a space is needed here so that the parser understandings this is an array, but I'd suggest adding some tags that help make the project searchable. I've added a few suggestions:

```suggestion
  - java
  - logging
```